pom.xml

```diff
@@ -1613,6 +1613,7 @@ under the License.
 								<exclude>org.apache.flink.api.scala.hadoop.mapred.HadoopOutputFormat</exclude>
 								<exclude>org.apache.flink.api.scala.hadoop.mapreduce.HadoopInputFormat</exclude>
 								<exclude>org.apache.flink.api.scala.hadoop.mapreduce.HadoopOutputFormat</exclude>
+								<exclude>org.apache.flink.api.scala.ExecutionEnvironment</exclude>
```

Comment:
I temporarily added it to the exclusion list so that Travis can continue compiling and verifying other changes. cc @tillrohrmann and @StephanEwen

Reply:
Hmm, ideally we don't exclude `ExecutionEnvironment` permanently. Is there maybe an annotation of more fine grained configuration option for the `japicmp-maven-plugin`?

Reply:
I just tried mark those changed API with annotation (Experimental and PublicEvolving) but still report incompatibility error (the former annotation reports `METHOD_RETURN_TYPE_CHANGED` and the second annotation reports `METHOD_REMOVED`). Just like the PR #6266 .

Reply:
However, existing programs that use these APIs are compatible. But the japicmp always reports  error.

Reply:
Would `org.apache.flink.api.scala.ExecutionEnvironment#createInput` work as an exclusion? This is then more specific and would not exclude the complete `ExecutionEnvironment`.

The danger is still that we would not detect any breaking API changes made to these methods in the future. Could we maybe add a manual test which fails if this happens and add a note stating why we have these tests?

Reply:
@tillrohrmann  Before I change it, do I want to hear your opinion (adding a reflection-based manual test?) remains the same?